openerp-atomfeed-service/src/main/java/org/bahmni/feed/openerp/domain/OpenMRSResource.java

```diff
@@ -0,0 +1,52 @@
+package org.bahmni.feed.openerp.domain;
+
+import java.util.Date;
+
+public class OpenMRSResource {
+
```

Comment:
also introduce a field for the resource, for representing the resource properties from openmrs. 

private Map<String, String> properties;